### PR TITLE
DOC: add form validation

### PR DESCRIPTION
**Pull request by zonky2**
DOC: add form validation

### CODE REVIEW

File: docs/Form/Form.md [hunk 1]
```diff
@@ -187,4 +187,33 @@ whatever you like.
         // The model will now contain the changes so you can save it
         $objModel->save();
     }
+```
+
+### Add field validation
+You can add your own field validation per `addValidator` or directly in your field per `save_callback`.
```
Comment (Toflar) [inline docs/Form/Form.md]: Replace `per` by `using`. `per` in this context sounds too German too me ;-)

File: docs/Form/Form.md [hunk 2]
```diff
@@ -187,4 +187,33 @@ whatever you like.
...
+
+```php
+<?php
+    // use addValidator
```
Comment (qzminski) [inline docs/Form/Form.md]: No need for comment here.

File: docs/Form/Form.md [hunk 3]
```diff
@@ -187,4 +187,33 @@ whatever you like.
...
+        if ($varValue == 'xxx@yyy.zz') {
+            throw new \Exception('Wrong email address!');
+        }
+    
+        return $varValue;
+    });
+    
+    // or directly in your field
+    $objForm->addFormField('email', [
+        // Feldeinstellungen
```
Comment (Toflar) [inline docs/Form/Form.md]: Please replace the German comment by an English one.

File: docs/Form/Form.md [hunk 4]
```diff
@@ -187,4 +187,33 @@ whatever you like.
...
+```
+
+### Add field validation
+You can add your own field validation per `addValidator` or directly in your field per `save_callback`.
+
+```php
+<?php
+    // use addValidator
+    $objForm->addValidator('email', function($varValue, \Widget $objWidget, Form $objForm){
+        if ($varValue == 'xxx@yyy.zz') {
```
Comment (qzminski) [inline docs/Form/Form.md]: I think it would rather make more sense to use `!==` operator here...

File: docs/Form/Form.md (hunk 3 above)
Comment (qzminski) [inline docs/Form/Form.md]: This comment can be dropped.

**Comment by zonky2** [comment]
now o.k.?

File: docs/Form/Form.md (hunk 1 above)
Comment (zonky2) [reply docs/Form/Form.md]: I had change `per` to `using` ...

**Pull request merged**